string object_type
 ---
string recognizer_name